src/org/labkey/targetedms/chart/ChromatogramDataset.java

```diff
@@ -525,25 +524,31 @@ private PeakInChart addPrecursorAsSeries(XYSeriesCollection dataset, Chromatogra
             double minTime = chromatogramRtRange.getMinRt();
             double maxTime = chromatogramRtRange.getMaxRt();
 
-            Set<Integer> transitionChromIndexes = TransitionManager.getTransitionChromatogramIndexes(pChromInfo.getId());
+            Map<Integer, Boolean> transitionChromIndexes = TransitionManager.getTransitionChromatogramIndexes(pChromInfo.getId());
+
+            // If none of the transitions are quantitative we will add them all.  Otherwise add only quantitative transitions.
+            boolean addAll = !transitionChromIndexes.entrySet().stream().anyMatch(k -> !Boolean.FALSE.equals(k.getValue()));
```

Comment:
I'm getting lost in the ! operators and it being equal to false. Is this because we may have null values in the map? Could this be:

```suggestion
            boolean addAll = transitionChromIndexes.values().stream().allMatch(v -> v == null || v.booleanValue()));
```

Reply:
Yes, the map can have null values.   Quantitative transitions do not have the "quantitative" attribute in Skyline XML so those are all null in the database.  It is hopefully a little clearer in this PR for 20.7: https://github.com/LabKey/targetedms/pull/281